### PR TITLE
Register Section

### PR DESCRIPTION
**Pull request by AyeshnaSingh**
Completed the issue #6 
- Added the register section
  -  Button
  -  Svg
  -  Title
  -  Paragraph 

Used HTML CSS JavaScript React

### CODE REVIEW

File: src/components/RegisterSection/registerSection.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:1]:
```suggestion
```

File: src/components/RegisterSection/registerSection.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+function RegisterSection() {
+  return (
+    <div className="min-h-screen flex items-center justify-center p-16">
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:5]:
```suggestion
    <div className="flex items-center justify-center p-16">
```

File: src/components/RegisterSection/registerSection.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="bg-white bg-opacity-5 backdrop-blur border border-white border-opacity-10 h-screen w-screen m-12 rounded-lg text-center text-white relative p-4 sm:p-6 md:p-8 lg:p-12 flex flex-col justify-center md:flex-row">
+
+        <div className="flex-grow md:w-1/3" />
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:8]: What is its _use-case_

File: src/components/RegisterSection/registerSection.jsx (hunk 2 above)
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:5]:
```suggestion
    <div className="flex items-center justify-center p-16">
```

File: src/components/RegisterSection/registerSection.jsx (hunk 3 above)
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:8]:
```suggestion
        
```

What is its _use-case_?

File: src/globals.css
Comment (shlok-2003) [inline src/globals.css:1]: add these css files in `tailwind.config.js`

File: src/components/RegisterSection/registerSection.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+        <div className="md:w-1/3 flex flex-col items-center">
+          <h1 className="text-2xl sm:text-3xl lg:text-4xl font-semibold mb-2 spline-sans title-color">Find out about</h1>
+          <h1 className="text-2xl sm:text-3xl lg:text-4xl font-semibold mb-2 spline-sans title-color">Hacktober Fest</h1>
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:12]: '**Find out about Hacktoberfest**' must be inside a single container

File: src/globals.css [hunk 5]
```diff
@@ -9,3 +9,24 @@
...
+
+.button-bg {
+    background-color: rgb(76, 88, 254);
+}
+
+.spline-sans {
+    font-family: 'Spline Sans', sans-serif;
+}
+
+.title-color{
```
Comment (shlok-2003) [inline src/globals.css:22]: Give a more defining name 
For example: `title-gray-150`

File: src/components/RegisterSection/registerSection.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+          <p className="text-gray-300 text-xs sm:text-sm md:text-base lg:text-lg">
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:13]:
```suggestion
          <p className="text-gray-300 text-xs sm:text-sm md:text-base">
```

File: src/components/RegisterSection/registerSection.jsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+            Lorem ipsum dolor sit amet consectetur. Tellus dignissim amet ultricies lobortis nunc egestas amet id.
+          </p>
+          <button className="button-bg text-white rounded-lg z-10 py-2 px-4 mt-4 sm:mt-6 md:mt-8 lg:mt-10 mx-auto block text-sm sm:text-base lg:text-lg">Register Now</button>
+        </div>
+
+        <div className="flex-grow md:w-1/3" />
+
+        <div className="absolute bottom-0 items-center w-fit h-auto bg-blend-overlay bg-no-repeat" style={{ zIndex: -1 }}>
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:21]:
```suggestion
        <div className="absolute bottom-0 items-center w-fit h-auto bg-blend-overlay bg-no-repeat -z-10">
```

File: src/components/RegisterSection/registerSection.jsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+        <div className="md:w-1/3 flex flex-col items-center">
+          <h1 className="text-2xl sm:text-3xl lg:text-4xl font-semibold mb-2 spline-sans title-color">Find out about</h1>
+          <h1 className="text-2xl sm:text-3xl lg:text-4xl font-semibold mb-2 spline-sans title-color">Hacktober Fest</h1>
+          <p className="text-gray-300 text-xs sm:text-sm md:text-base lg:text-lg">
+            Lorem ipsum dolor sit amet consectetur. Tellus dignissim amet ultricies lobortis nunc egestas amet id.
+          </p>
+          <button className="button-bg text-white rounded-lg z-10 py-2 px-4 mt-4 sm:mt-6 md:mt-8 lg:mt-10 mx-auto block text-sm sm:text-base lg:text-lg">Register Now</button>
+        </div>
+
+        <div className="flex-grow md:w-1/3" />
```
Comment (shlok-2003) [inline src/components/RegisterSection/registerSection.jsx:19]:
```suggestion
        
```
What is its _use-case_?

**Comment by Shivam-Fl** [comment]
- used the svg as background image to make it responsive 
- Removed 5-6 additional useless div

**Comment by Shivam-Fl** [comment]
- Removed all css from global.css
- Imported spline-sans

**Pull request merged**